### DISCUSSION TITLE
better structures

### DISCUSSION THREAD

**Discussion by Herobrine24033** [jacobfortune/Songs-Of-War-Addon#3] [Ideas]
So i love the mod but the only thing i hated about it was the structures like for the ardoni structures was boring in the series there villages are actually village not three huts and adaraxia was a floating city not three huts. I think that the ardoni villages should be bigger and some floating. the felina villages look good but they need to be bigger. And magnerite villages are just pathetic remake the magnerite villages to the town senn met insigneis. And there are no structures for humans. And add the capitals. I get that i could download the ardonia map its more fun to do a random generated world and explore new things instead of knowing where everthing is 
 @ThetruePyrobyte if you see this i have a bunch more ideas reach out to me on my email gearsofwar299@outlook.com or my Discord @herobrine24033 sorry if this message sounds like im yelling i made this in full respect

**Comment by lesleyespire** [comment]
Wow, thanks for replying. It is true that the structures are very ugly sometimes and aren’t very populated, also a lot of the addon is broken at the moment because I am trying to add jaggathan races, which in turn broke all the races. You can now obtain songs and you get other abilities but only once jaggathan stops breaking everything. Finally, I’m not very good at the code that makes structures but I will definitely do my best to improve it but I might not be able too.

Thanks for your feedback and compliments. (;
I appreciate it
Pyrobyte,

Sent from my iPhone

On 5 Jul 2023, at 5:03 am, Herobrine24033 ***@***.***> wrote:

﻿

So i love the mod but the only thing i hated about it was the structures like for the ardoni structures was boring in the series there villages are actually village not three huts and adaraxia was a floating city not three huts. I think that the ardoni villages should be bigger and some floating. the felina villages look good but they need to be bigger. And magnerite villages are just pathetic remake the magnerite villages to the town senn met insigneis. And there are no structures for humans. And add the capitals. I get that i could download the ardonia map its more fun to do a random generated world and explore new things instead of knowing where everthing is
